### DISCUSSION TITLE
Kernel Head in RTMDET

### DISCUSSION THREAD

**Discussion by GTrui6** [open-mmlab/mmdetection#10032] [General]
Why the kernel head in RTMDET has 169 dimensions? The paper says:"The vector is divided into three parts (lengths are 88, 72, and 9 respectively), which are used to form the kernels of three dynamic convolution layers". So why is that?

**Comment by hhaAndroid** [comment]
@GTrui6 Its a reference from condinst, you can see the original paper

**Reply by GTrui6** [reply]
Thanks for your reply, i have saw the paper
